src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
It is necessary since the current implementation is hacky. The recording is starting before it should (with enter) and it has to be filtered with `first`.